### PR TITLE
Feature/migration to new repo

### PR DESCRIPTION
**Pull request by raphaellueckl**
# Merge Checklist:
- [ ] Code selfreview has been performed ([Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)).
- [ ] Tests are sufficient.
- [ ] Modifications still work in IE.
- [ ] Documentation is up to date.
- [ ] Changelog is up to date.
- [ ] Typescript typings for properties are up to date.
- [ ] Designers approved changes or no approval needed.
- [ ] Dependencies to other components still work.

### CODE REVIEW

**Comment by 01231** [comment]
@raphaellueckl 
Some things I've noticed we could do:

- changing `patterns-lib-devs` to `pattern-lib-devs` (slack channel)
- rename patterns-library-polyfill to `patterns-library-polyfill` for consistency
- inform devs about our name change

What do you think?

File: ARCHITECTURE.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Elements](https://developers.google.com/web/fundamentals/web-components/customel
 (short: **CE**), a browser-native technology supported by [web
 standards](https://html.spec.whatwg.org/multipage/custom-elements.html) and poly-fillable for older browsers.
 
-[Here](https://raw.githubusercontent.com/axa-ch/patterns-library/develop/FOUNDATION.md) you can read how we arrived at this choice.
+[Here](https://raw.githubusercontent.com/axa-ch-/patterns-library/develop/FOUNDATION.md) you can read how we arrived at this choice.
```
Comment (01231) [inline ARCHITECTURE.md]: this link is broken

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@axa-ch/patterns-library",
+  "name": "@axa-ch/pattern-library",
```
Comment (01231) [inline package.json:2]: shouldn't it be `@axa-ch/patterns-library`?

**Review by 01231 — COMMENTED** [review]
small stuff

File: ARCHITECTURE.md (hunk 1 above)
Comment (01231) [reply ARCHITECTURE.md]: fixed it

File: package.json (hunk 2 above)
Comment (raphaellueckl) [reply package.json:2]: the currently published version of the PL is 1.XX and not ours, so it doesn't really matter. For consistency I changed the name here. I does not habe an impact anyways.

**Pull request merged**